exercises/knapsack/description.md

```diff
@@ -0,0 +1,28 @@
+In this exercise, lets try to solve a classic problem.
+
+Bob is a thief. After months of planning, he 
+finally manages to crack the security systems of a 
+high-class apartment. In front of him are many 
+items, each with a value (v) and weight (w). Bob of
+course wants to maximise the total value he can 
+get; he would of course take all the items if he
+could. However, to his horrer, he realised that the 
+knapsack he carried with him can only hold a weight W.
+
+Given a knapsack of weight W, can you help Bob 
+determine the maximum value he can get from the 
+items in the house? Note, you can take only one of 
+each item. 
+
+All values given will be strictly positive integers.
+Items will be represented as a list of pairs (wi, vi) where the first element (wi) is the weight of the ith item and vi is the value for that item.
```

Comment:
```suggestion
Items will be represented as a list of pairs [wi, vi] where the first element (wi) is the weight of the ith item and (vi) is the value for that item.
```

Reply:
@rpottsoh: I proposed to change this into

    a list of pairs, `wi` and `vi`, ...

to avoid concrete syntax for pairs.